ambry-api/src/main/java/com/github/ambry/named/NamedBlobDb.java

```diff
@@ -70,10 +70,11 @@ CompletableFuture<Page<NamedBlobRecord>> list(String accountName, String contain
    * @param record the {@link NamedBlobRecord}
    * @param state the {@link NamedBlobState}
    * @param isUpsert the {@link Boolean}
+   * @param isTtlUpdate the {@link Boolean}
    * @return a {@link CompletableFuture} that will eventually contain a {@link PutResult} or an exception if an error
    *         occurred.
    */
-  CompletableFuture<PutResult> put(NamedBlobRecord record, NamedBlobState state, Boolean isUpsert);
+  CompletableFuture<PutResult> put(NamedBlobRecord record, NamedBlobState state, Boolean isUpsert, Boolean isTtlUpdate);
```

Comment:
can we not reuse put method here, can we just create a unique method for updating blob state? something like
`CompletableFuture<PutResult> updateBlobState(NamedBlobRecord record, NamedBlobState state)`?

Reply:
Good point, i just updated to use ```updateBlobStateToReady```  instead of ```put```